CMakeLists.txt

```diff
@@ -7,6 +7,8 @@ cmake_policy(SET CMP0009 NEW)
 
 project(aktualizr)
 
+include_directories(/usr/include/ostree-1)
```

Comment:
Is this needed? We have this line later: `include_directories(${LIBOSTREE_INCLUDE_DIRS})`

Reply:
Please check this cmake module
https://github.com/advancedtelematic/aktualizr/blob/78e19207414a2373d70d63ae5a4810f9b956cbd8/cmake-modules/FindOSTree.cmake#L1

Normally we install ostree in default path "/usr" so the include dir will be located on /usr/include/ostree-1 and lib in /usr/lib!

Not sure this module can properly find the include directories since it nested to ostree-1, for *.so file it works because it located in default path for the linker

Reply:
That's what I getting!

In file included from /home/kbushko/work/pr/feat_OTA-2753_CI_ubuntu_xenial_build_tests/aktualizr/tests/aktualizr_cycle_simple.cc:10:0:
/home/kbushko/work/pr/feat_OTA-2753_CI_ubuntu_xenial_build_tests/aktualizr/src/libaktualizr/package_manager/ostreemanager.h:8:10: fatal error: ostree.h: No such file or directory
 #include <ostree.h>
          ^~~~~~~~~~

Reply:
Ah, this is not an issue with FindOSTree.cmake, the problem is in the aktualizr_cycle_simpl.cc this test include ostreemanager.h which includes ostree.h and if we build project without OSTREE it will fail since find_package for OSTree is not executed!

This actually is a case when TEST which executed by default has dependencies on the package which is optional, in this particular case -DBUILD_OSTREE